csp/gitprojectsettings.csp

```diff
@@ -403,6 +435,21 @@ body {
             </div>
         </div>
 
+        <div class="form-group row mb-3">
+            <label for="basicMode" class="offset-sm-1 col-sm-3 col-form-label" data-toggle="tooltip" data-placement="top" title="Enable basic mode, greatly simplifying the Git Source Control package, requiring no knowledge of git">Basic Mode</label>
```

Comment:
Reword:
Enable basic mode, which reduces the number of menu options and requires less knowledge of git. This is suitable for a single-developer (non-shared) namespace.

Reply:
We need to think about the interaction of this with system-wide. This is really tri-state and should be represented via a radio button set: enabled, disabled, default (follows system mode and corresponds to unset global) - this will need changes in the settings methods too.